src/txHashBuilder_test.c

```diff
@@ -330,13 +336,16 @@ void run_txHashBuilder_test()
 	ITERATE(it, outputs) {
 		uint8_t tmp[70];
 		size_t tmpSize = decode_hex(PTR_PIC(it->rawAddressHex), tmp, SIZEOF(tmp));
-		txHashBuilder_addOutput(
+		txHashBuilder_addOutput_basicData(
 		        &builder,
 		        tmp, tmpSize,
-		        it->amount
+		        it->amount,
+		        0
 		);
 	}
 
+	// TODO multiasset output? covered in integration tests
```

Comment:
what's the issue with adding a sample multiasset output here for the sake of completeness?

Reply:
I have added it. I was afraid it might consume too much memory, but I have reused buffers and refactored it a tiny bit and no problem appeared.